packages/google_maps_flutter/google_maps_flutter_ios/example/ios14/ios/RunnerTests/FLTGoogleMapJSONConversionsConversionTests.m

```diff
@@ -226,115 +225,165 @@ - (void)testCameraUpdateFromArrayNewCameraPosition {
+  [[classMockCameraUpdate expect] zoomBy:zoom];
 
-  NSArray *channelValueWithPoint = @[ @"zoomBy", @1, @[ @2, @3 ] ];
+  const CGFloat x = 2;
+  const CGFloat y = 3;
+  FGMPlatformCameraUpdateZoomBy *platformUpdate =
+      [FGMPlatformCameraUpdateZoomBy makeWithAmount:zoom focus:[FGMPlatformPoint makeWithX:x y:y]];
 
-  [FLTGoogleMapJSONConversions cameraUpdateFromArray:channelValueWithPoint];
+  GMSCameraUpdate *update = FGMGetCameraUpdateForPigeonCameraUpdate(
+      [FGMPlatformCameraUpdate makeWithCameraUpdate:platformUpdate]);
 
-  [[classMockCameraUpdate expect] zoomBy:1 atPoint:CGPointMake(2, 3)];
+  [[classMockCameraUpdate expect] zoomBy:zoom atPoint:CGPointMake(x, y)];
   [classMockCameraUpdate stopMocking];
 }
 
-- (void)testCameraUpdateFromArrayZoomIn {
-  NSArray *channelValueNoPoint = @[ @"zoomIn" ];
+- (void)testCameraUpdateFromZoomIn {
+  FGMPlatformCameraUpdateZoom *platformUpdate = [FGMPlatformCameraUpdateZoom makeWithOut:NO];
 
   id classMockCameraUpdate = OCMClassMock([GMSCameraUpdate class]);
-  [FLTGoogleMapJSONConversions cameraUpdateFromArray:channelValueNoPoint];
+  GMSCameraUpdate *update = FGMGetCameraUpdateForPigeonCameraUpdate(
+      [FGMPlatformCameraUpdate makeWithCameraUpdate:platformUpdate]);
 
   [[classMockCameraUpdate expect] zoomIn];
   [classMockCameraUpdate stopMocking];
 }
 
-- (void)testCameraUpdateFromArrayZoomOut {
-  NSArray *channelValueNoPoint = @[ @"zoomOut" ];
+- (void)testCameraUpdateFromZoomOut {
+  FGMPlatformCameraUpdateZoom *platformUpdate = [FGMPlatformCameraUpdateZoom makeWithOut:YES];
 
   id classMockCameraUpdate = OCMClassMock([GMSCameraUpdate class]);
-  [FLTGoogleMapJSONConversions cameraUpdateFromArray:channelValueNoPoint];
+  GMSCameraUpdate *update = FGMGetCameraUpdateForPigeonCameraUpdate(
+      [FGMPlatformCameraUpdate makeWithCameraUpdate:platformUpdate]);
 
   [[classMockCameraUpdate expect] zoomOut];
   [classMockCameraUpdate stopMocking];
 }
 
-- (void)testCameraUpdateFromArrayZoomTo {
-  NSArray *channelValueNoPoint = @[ @"zoomTo", @1 ];
+- (void)testCameraUpdateFromZoomTo {
+  const CGFloat zoom = 1;
+  FGMPlatformCameraUpdateZoomTo *platformUpdate = [FGMPlatformCameraUpdateZoomTo makeWithZoom:zoom];
 
   id classMockCameraUpdate = OCMClassMock([GMSCameraUpdate class]);
-  [FLTGoogleMapJSONConversions cameraUpdateFromArray:channelValueNoPoint];
+  GMSCameraUpdate *update = FGMGetCameraUpdateForPigeonCameraUpdate(
+      [FGMPlatformCameraUpdate makeWithCameraUpdate:platformUpdate]);
 
-  [[classMockCameraUpdate expect] zoomTo:1];
+  [[classMockCameraUpdate expect] zoomTo:zoom];
   [classMockCameraUpdate stopMocking];
 }
 
+- (void)testStrokeStylesFromPatterns {
+  NSArray<FGMPlatformPatternItem *> *patterns = @[
+    [FGMPlatformPatternItem makeWithType:FGMPlatformPatternItemTypeGap length:@(1)],
+    [FGMPlatformPatternItem makeWithType:FGMPlatformPatternItemTypeDash length:@(1)]
+  ];
+  UIColor *strokeColor = [UIColor redColor];
```

Comment:
Opportunistic `UIColor.redColor` here? 😉

Reply:
Done. I also found all the other `UIColor` fixed colors and updated them to the new style.